### ISSUE TITLE
Links to plone static pages from within data-adh-html-translate not working on production/staging

### ISSUE THREAD

**Issue by localgrr** [liqd/adhocracy3#2011]
in mercator/2016/Proposal/Create.html we have the following string
```
<span data-adh-html-translate="TR__MERCATOR_PROPOSAL_STRENGTHEN_ANNOTATION" data-translate-templates="{
                    link: '&lt;a data-ng-href=&quot;\\{\\{ selection_criteria_link \\}\\}&quot; target=&quot;_blank&quot;&gt;{{ content }}&lt;/a&gt;'
                    }"></span>
```

the selection_criteria_link is not working on production/staging but does work locally for unknown reasons. I have hard-coded this link as we go live today

**Comment by local-girl** [comment]
It's possible this was because I had the link as /en/idea-space/selection-criteria/ instead of https://advocate-europe.eu/en/idea-space/selection-criteria/ - no time to test it now

**Comment by xi** [comment]
Fixed by 212e148
